site/static/docs/4.3/assets/js/validate-forms.js

```diff
@@ -0,0 +1,20 @@
+// Example starter JavaScript for disabling form submissions if there are invalid fields
+(function () {
+  'use strict'
+
+  // Fetch all the forms we want to apply custom Bootstrap validation styles to
+  var forms = document.querySelectorAll('.needs-validation')
+
+  // Loop over them and prevent submission
+  Array.prototype.slice.call(forms)
```

Comment:
slice isn't needed here since we are in our v5 doc

Reply:
This is unrelated to this patch, though :)

I don't change the snippet, I just fix the docs.